.github/workflows/system-tests.yml

```diff
@@ -33,6 +33,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: 'DataDog/system-tests'
+          ref: conti/dsm-manual-checkpoint-tests
```

Comment:
Should this still be pointing at this branch?

Reply:
removed this